definitions.yaml

```diff
@@ -0,0 +1,356 @@
+      authorized_cidrs:
+        description: |-
+          Allowed source IPs or CIDRs for this service.
+        type: array
+        items:
+          $ref: "#/definitions/CIDR"
+      authorized_recipients:
+        description: |-
+          If non empty, the service will be able
+          to send messages only to these fiscal codes.
+        type: array
+        items:
+          $ref: "#/definitions/FiscalCode"
+      is_visible:
+        type: boolean
+        default: false
+      max_allowed_payment_amount:
+        $ref: "#/definitions/MaxAllowedPaymentAmount"
+      organization_fiscal_code:
+        $ref: "#/definitions/OrganizationFiscalCode"
+      version:
+        type: integer
+      id:
+        type: string
+    required:
+      - service_id
+      - service_name
+      - organization_name
+      - department_name
+      - authorized_cidrs
+      - authorized_recipients
+      - organization_fiscal_code
+
+  #Returned by PUT /services/:serviceId
+  ServicePublic:
+    title: Service (public)
+    description: A Service associated to an user's subscription.
+    type: object
+    properties:
+      service_id:
+        $ref: "#/definitions/ServiceId"
+      service_name:
+        $ref: "#/definitions/ServiceName"
+      organization_name:
+        $ref: "#/definitions/OrganizationName"
+      department_name:
+        $ref: "#/definitions/DepartmentName"
+      organization_fiscal_code:
+        $ref: "#/definitions/OrganizationFiscalCode"
+      version:
+        type: integer
+    required:
+      - service_id
+      - service_name
+      - organization_name
+      - department_name
+      - organization_fiscal_code
+      - version
+
+  ServiceId:
+    type: string
+    description: |-
+      The ID of the Service. Equals the subscriptionId of a registered
+      API user.
+    minLength: 1
+
+  ServiceName:
+    type: string
+    description: The name of the service. Will be added to the content of sent messages.
+    minLength: 1
+
+  OrganizationName:
+    type: string
+    description: |-
+      The organization that runs the service. Will be added to the content
+      of sent messages to identify the sender.
+    minLength: 1
+
+  DepartmentName:
+    type: string
+    description: |-
+      The department inside the organization that runs the service. Will
+      be added to the content of sent messages.
+    minLength: 1
+
+  CIDR:
+    type: string
+    description: Describes a single IP or a range of IPs.
+    pattern: "^([0-9]{1,3}[.]){3}[0-9]{1,3}(/([0-9]|[1-2][0-9]|3[0-2]))?$"
+
+  FiscalCode:
+    type: string
+    description: User's fiscal code.
+    format: FiscalCode
+    x-import: italia-ts-commons/lib/strings
+    example: SPNDNL80R13C555X
+
+  OrganizationFiscalCode:
+    type: string
+    description: Organization fiscal code.
+    format: OrganizationFiscalCode
+    x-import: italia-ts-commons/lib/strings
+    example: "12345678901"
+
+  MaxAllowedPaymentAmount:
```

Comment:
can we reference [this](https://github.com/teamdigitale/io-functions/blob/master/api/definitions.yaml#L134) instead of redeclaring it?